tests/unit/dispatchers/dispatchersService.test.ts

```diff
@@ -73,6 +73,16 @@ describe('Dispatchers Service', () => {
     expect(res.id).toBe(dispatcherData.id);
   });
 
+  it('should update a dispatcher', async () => {
+    nock(`${host}${DISPATCHERS_BASE}`).put(`/${dispatcherId}`).reply(200, {
```

Comment:
Sheesh, when I look at this, this is crazy to read, we should probably look at doing this better in the future.